src/rabbit_prelaunch.erl

```diff
@@ -151,11 +153,24 @@ dist_port_use_check_ipv6(NodeHost, Port) ->
                                          no_return().
 
 dist_port_use_check_fail(Port, Host) ->
-    {ok, Names} = rabbit_nodes:names(Host),
+    io:format("dist_port_use_check_fail port:~p host:~p~n",[Port, Host]),
+    {ok, Names} = empd_names(Host),
     case [N || {N, P} <- Names, P =:= Port] of
         []     -> io:format("ERROR: distribution port ~b in use on ~s "
                             "(by non-Erlang process?)~n", [Port, Host]);
         [Name] -> io:format("ERROR: distribution port ~b in use by ~s@~s~n",
                             [Port, Name, Host])
     end,
     rabbit_misc:quit(?ERROR_CODE).
+
+empd_names(NodeHost) ->
+   case rabbit_nodes:names(NodeHost) of
+       {error, EpmdReason} ->
+           io:format("ERROR: epmd error for host ~s: ~s, trying with ipv6~n",
+               [NodeHost, rabbit_misc:format_inet_error(EpmdReason)]),
+           rabbit_nodes:names(?LOCAL_IPV6_ADDRESS);
```

Comment:
Could you please add a comment explaning the problem solved/worked around here?

Reply:
@dumbbell did it.
I left `empd_names(NodeHost)` inside  `rabbit_prelaunch.erl`. Please let me know if you want to move also this function to  `rabbit_networking`